### ISSUE TITLE
about creat the ground-truth files..

### ISSUE THREAD

**Issue by HoldYourFire** [Cartucho/mAP#46]
eg:the following format ：
<class_name> <left> <top> <right> <bottom>，if I use my own images ,How do I get the coordinates like yours?Could you explain ?
Thank you.

**Comment by HoldYourFire** [comment]
@Cartucho Thank you for being so quick.I am using the detection of keras-yolov3.I have trained the model, and it has only one class（person）.The sample data format is VOC2007.Uh.....what should I do next？So I can get my P-R curve(AP).
Looking forward to your reply

**Comment by HoldYourFire** [comment]
Oh I use the code from https://github.com/qqwweee/keras-yolo3 @Cartucho

**Comment by Cartucho** [comment]
Great. You can find the instructions for keras in the README inside the `extra/`directory.

**Comment by HoldYourFire** [comment]
@Cartucho sorry to bother you again.I see the the python script: python3 convert_keras-yolo3.py.I thought it needs the annotation file that refers to ground-truth in (keras-yolo3 format) which are produced by labelImg(am i right?...). But what about the  annotation file that refers to predictions.I don't have it ...do you know ?Or am I getting it wrong?

**Comment by HoldYourFire** [comment]
@Cartucho uh the code in convert-keras-yolo3.py line 38 -"help="The annotation file that refers to ground-truth in (keras-yolo3 format)")"what's the meaning of this?........Could you explain it? Is this also a txt file?

**Comment by Cartucho** [comment]
I'm not really sure, those codes were added by @gustavovaliati , perhaps he can help you.

Anyway, according to the keras README, the files are in this format `image_file_path box1 ... boxN`:
e.g.: path/to/img1.jpg 50,100,150,200,0 30,50,200,120,3

When you train you have a training set and a test set. The test set is your ground-truth.

**Comment by gustavovaliati** [comment]
Hi @HoldYourFire,

The script `extra/convert_keras-yolo3.py` has been created to handle the annotations according to the format used by https://github.com/qqwweee/keras-yolo3.

In such way, the argument `--gt` expects an annotation file for the grounding truth in the format of documented in https://github.com/qqwweee/keras-yolo3#training, as @Cartucho stated.

**Comment by HoldYourFire** [comment]
@gustavovaliati thank you very much.Luckily,I solved the problem.by the way,the code is so pwerful!!Thank you for your reply.
